Robot Code/src/org/usfirst/frc/team3316/robot/sequences/AutonomousSequence.java

```diff
@@ -2,9 +2,14 @@
 //TODO: Move this to a new package of autonomous sequences.
 
 import org.usfirst.frc.team3316.robot.commands.DBugCommandGroup;
+import org.usfirst.frc.team3316.robot.commands.chassis.auton.CrossDefense;
 
 public class AutonomousSequence extends DBugCommandGroup {
 	public AutonomousSequence() {
-		
+		addSequential(new CrossDefense(false));
```

Comment:
Consider adding an enum of Direction  (Forwards & Backwards) in order to replace the boolean and clarify the meaning of "new CrossDefense(false)"